src/wp-admin/includes/file.php

```diff
@@ -1967,22 +1967,22 @@ function _unzip_file_pclzip( $file, $to, $needed_dirs = array() ) {
 	}
 
 	// Extract the files from the zip.
-	foreach ( $archive_files as $file ) {
+	foreach ( $archive_files as $archive_file ) {
 		if ( $file['folder'] ) {
```

Comment:
```suggestion
		if ( $archive_file['folder'] ) {
```

Missing instance of old variable

Reply:
Oh whoops! Good spot